app/assets/stylesheets/application.scss

```diff
@@ -2,3 +2,9 @@
 @import "dashboard";
 @import "bootstrap";
 @import "static_pages";
+@import "font-awesome";
+
+
+.bordless {
+  border: none;
```

Comment:
przenieś to może do custom.scss, albo czegoś takiego. Wtedy jest porządek większy w application.scss